src/adapters/http/preLoad/router.ts

```diff
@@ -1,28 +1,34 @@
 import express from 'express';
 import { pipe } from 'fp-ts/lib/function';
 import * as E from 'fp-ts/Either';
-import * as utils from '../utils';
+import * as T from 'fp-ts/Task';
+import * as TE from 'fp-ts/TaskEither';
 import { PreLoadUseCase } from '../../../useCases/PreLoadUseCase';
 import { PreLoadRequestBody } from '../../../generated/definitions/PreLoadRequestBody';
 import { ApiKey } from '../../../generated/definitions/ApiKey';
+import { makeProblem } from '../codec';
+import { Handler, toExpressHandler } from '../Handler';
 
 const preloadHandler =
-  (preLoadUseCase: PreLoadUseCase): express.Handler =>
+  (preLoadUseCase: PreLoadUseCase): Handler =>
   (req, res) =>
     pipe(
       E.of(preLoadUseCase),
       E.ap(ApiKey.decode(req.headers['x-api-key'])),
       E.ap(PreLoadRequestBody.decode(req.body)),
-      utils.sendResponse(res)(
-        (_error) => res.status(500).send(utils.makeAPIProblem(500, 'Internal Server Error')(undefined)),
-        (_) => res.status(_.statusCode).send(_.returned)
+      // Create response
+      E.map(
+        TE.fold(
+          (_) => T.of(res.status(500).send(makeProblem(500))),
+          (_) => T.of(res.status(_.statusCode).send(_.returned))
+        )
```

Comment:
I would uniform the way we are going to send the response, so:
- we use the `TE.fold -> T.of`
- we use the `flow -> TE.bimap -> TE.union` form

Reply:
Of course! Do you have any preference between the two? If you don't I will pick one based on my preferences :)

Reply:
I have a small preference for the `TE.fold -> T.of` form

Reply:
Done :cowboy_hat_face: